src/m365/outlook/commands/roomlist/roomlist-list.ts

```diff
@@ -0,0 +1,45 @@
+import { Logger } from '../../../../cli';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: GlobalOptions;
+}
+
+class OutlookRoomlistListCommand extends GraphCommand {
```

Comment:
Since this is a list command we recommend inheriting from the GraphItemsListCommand base class. This allows you to traverse over all paged items in a list. You find an example under serviceannouncement-message-list

Reply:
Modified to code to use GraphItemListCommand. Thank you!